polymer-redux.html

```diff
@@ -1,2 +1,2 @@
-<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="./bower_components/polymer/polymer.html">
```

Comment:
it's best to keep this as a relative path like polymer team do in their elements. please remove.